application/src/main/java/uk/gov/hmcts/dm/actuate/health/WebChecker.java

```diff
@@ -34,7 +32,7 @@ private boolean getStatus() {
             final HealthCheckResponse healthCheckResponse = restTemplate.getForObject(url + "/health", HealthCheckResponse.class);
             return "UP".equalsIgnoreCase(healthCheckResponse.getStatus());
         } catch (Exception ex) {
-            log.error(name + " " + url + " Failed",keyValue("stackTrace",ex));
+            log.error(name + " " + url + "/health Failed", ex);
             return false;
```

Comment:
It looks like we could have 

LOG.error("{} {}/health Failed", name, url, ex);